packages/components/src/__future__/Select/_docs/Select.mdx

```diff
@@ -98,3 +98,23 @@ Add validation messages using `status` and `validationMessage`.
 
 Set `isFullWidth` to `true` to have the Select span the full width of its container.
 <Canvas of={SelectStories.FullWidth} />
+
+### Portals
+
+By default, the Select's popover will attach itself to the `body` of the document using React's `createPortal`.
+
+You can change the default behaviour by providing a `portalContainerId` to attach this to different element in the DOM. This can help to resolve issues that may arise with `z-index` or having a Select in a modal.
+
+<Canvas of={SelectStories.PortalContainer} />
+
+It is important to note that a `relative` position should be applied to container element that contains the portal, ie:
```

Comment:
I'd say "not absolute" rather than "is relative", as it will explain it clearer without the need for an example since everything is relative by default.

Reply:
But the container needs a `position: relative` for the popover to determine its position document since it uses `absolute`. Otherwise the static positioning on most elements could risk the Popover spilling out of the confines of it container.

Relative on container:
![Screenshot 2023-11-22 at 9 24 59 am](https://github.com/cultureamp/kaizen-design-system/assets/36558508/9d0b3578-4ece-41b9-8826-038a7f9ccb12)

Classless container:
![Screenshot 2023-11-22 at 9 24 53 am](https://github.com/cultureamp/kaizen-design-system/assets/36558508/bd49352b-17ac-43c4-9d9c-914955582fa3)

Reply:
Huh very curious, but do we want it to cut off..?

Reply:
I wouldn't cut it off as a default, but if someone wants to cut it off, I'm sure they can figure it out :) I think we could probably leave the advice off 😄

Reply:
We can remove it, but my thinking here is that this is a visual shorthand of how the portal container can effect the popover, instead of it just being on the body. 

I'm less beholden to having the advice around relative at the bottom since the example code can be seen in the canvas anyway.